### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by antoniogamiz**

### CODE REVIEW

**Comment by antoniogamiz** [comment]
@JJ Merge this when you can pls, the new version of Documentable depends on these changes.

File: t/011-external.t [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (JJ) [inline t/011-external.t:1]: Probably not needed...

File: t/120-templates.t [hunk 2]
```diff
@@ -19,7 +19,7 @@ The seven suspects are:
 =end pod
 
 stderr-like { $r = pod2html $=pod[0], :templates<templates> },
-            /'does not contain required templates'/,
+            /''/,
```
Comment (JJ) [inline t/120-templates.t:22]: Shouldn't this include some kind of error message? Any string will always contain the empty string...

**Comment by antoniogamiz** [comment]
Fixed.

**Pull request merged**